targettests/Remote-Sim/free_func.cpp

```diff
@@ -30,7 +32,11 @@ void ansatz(double theta) __qpu__ {
   cudaq::qvector q(2);
   x(q[0]);
   ry(theta, q[1]);
+#if CUDAQ_USE_STD20
```

Comment:
Same.

Reply:
Fixed in https://github.com/NVIDIA/cuda-quantum/pull/1435/commits/4ad7ecd84a4f0d7c8dac820095463a5234365331